### DISCUSSION TITLE
option for three finger swipe to move between drawings instead of frames?

### DISCUSSION THREAD

**Discussion by CowlBlue** [keiwando/toonsquid#576] [Ideas]
Hello!  I was wondering if it was possible to implement a feature to allow for three finger swipe to move between drawings instead of frames?  The feature already exists for the Forward/Backward buttons, but I really wish it existed for the three finger swipe, especially for larger projects in 24 fps where I am dealing with a variety of drawing durations (threes, twos, and fours).  

I much prefer the three finger swipe to flip between drawings because I find it much more comfortable to do so, and my current workaround lately is just to set the fps really low when I am doing my major keys, so I don't have to extend the frame duration and need to swipe many times or have to look at which drawing I am swiping to (if I allow the gesture to flip through multiple drawings).

**Comment by keiwando** [comment]
Yes, this is already on the todo list. Not sure when I'll get to it though.
Thanks for the feedback!

**Reply by CowlBlue** [reply]
Awesome!!

**Comment by CowlBlue** [comment]
I’m bumping this now that toonsquid 2.0 is out!  It’s the only feature I feel like is missing for me :).  Otherwise everything about toonsquid really blows other animation software out of the water!
